account_invoice_dispatch_option/views/res_partner_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.custom</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='accounting']" position="inside">
```

Comment:
```suggestion
            <xpath expr="//group[@name='accounting_entries']" position="after">         
```
The inside of accounting only shows commercial partner.
https://github.com/odoo/odoo/blob/ba1ba88874e2ae9716e59ec199eb739d58c3f771/addons/account/views/partner_view.xml#L197

Reply:
Sorry. I don't get your point. The group `accounting_entries` also exist under the page of accounting, isn't? Do you want me to show this field under another tab? https://github.com/odoo/odoo/blob/ba1ba88874e2ae9716e59ec199eb739d58c3f771/addons/account/views/partner_view.xml#L215

Reply:
I referred to this view. I apologize I did not check the details. 
https://github.com/qrtl/hls-custom/blob/12.0/account_partner_company_bank/views/res_partner_views.xml

I think this xpath can show all partners.
<xpath expr="//page[@name='accounting_disabled']/div" position="after">

Anyway, this field can be displayed all partner record.